### PR TITLE
chore(deps): dependency updates

### PR DESCRIPTION
**Pull request by JoshuaKGoldberg**
Updates of our npm dependencies that passed CI.

For this one we definitely don't want to merge in any major webpack breaking changes, so I might remove those...

### CODE REVIEW

File: packages/webpack-config/package.json [hunk 1]
```diff
@@ -39,15 +39,15 @@
...
     "node-sass": "^4.13.0",
     "postcss-flexbugs-fixes": "^4.1.0",
     "postcss-focus-visible": "^5.0.0",
     "postcss-loader": "^3.0.0",
-    "sass-loader": "^7.1.0",
-    "style-loader": "^1.2.0",
-    "terser-webpack-plugin": "^4.1.0",
+    "sass-loader": "^10.0.4",
+    "style-loader": "^2.0.0",
+    "terser-webpack-plugin": "^5.0.3",
```
Comment (JoshuaKGoldberg) [inline packages/webpack-config/package.json:50]: @jakemhiller these are the types of changes we can't do until Webpack 5, right?

(that's my understanding, and I'll block these until we get to 5 unless proven wrong)

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-icons@3.11.1-alpha.9b6659.0<br/>@codecademy/gamut-labs@5.4.2-alpha.9b6659.0<br/>@codecademy/gamut-styles@7.4.5-alpha.9b6659.0<br/>@codecademy/gamut-system@0.1.4-alpha.9b6659.0<br/>@codecademy/gamut@17.3.10-alpha.9b6659.0<br/>@codecademy/markdown-overrides@0.3.13-alpha.9b6659.0<br/>@codecademy/styleguide@18.5.10-alpha.9b6659.0<br/>@codecademy/webpack-config@5.2.1-alpha.9b6659.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5fb5c24790032e13c6a41187--gamut-preview.netlify.app>https://5fb5c24790032e13c6a41187--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/5fb5c24790032e13c6a41187>Deploy Logs</a></p>

File: packages/webpack-config/package.json (hunk 1 above)
Comment (jakemhiller) [reply packages/webpack-config/package.json:50]: Yeah, we don't want to update any of these

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-icons@3.11.1-alpha.2fc297.0<br/>@codecademy/gamut-labs@5.4.2-alpha.2fc297.0<br/>@codecademy/gamut-styles@7.4.5-alpha.2fc297.0<br/>@codecademy/gamut-system@0.1.4-alpha.2fc297.0<br/>@codecademy/gamut@17.3.10-alpha.2fc297.0<br/>@codecademy/markdown-overrides@0.3.13-alpha.2fc297.0<br/>@codecademy/styleguide@18.5.10-alpha.2fc297.0<br/>@codecademy/webpack-config@5.2.1-alpha.2fc297.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5fbba0223917fda25d11a518--gamut-preview.netlify.app>https://5fbba0223917fda25d11a518--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/5fbba0223917fda25d11a518>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-icons@3.11.1-alpha.5547cd.0<br/>@codecademy/gamut-labs@5.4.2-alpha.5547cd.0<br/>@codecademy/gamut-styles@7.4.5-alpha.5547cd.0<br/>@codecademy/gamut-system@0.1.4-alpha.5547cd.0<br/>@codecademy/gamut@17.3.10-alpha.5547cd.0<br/>@codecademy/markdown-overrides@0.3.13-alpha.5547cd.0<br/>@codecademy/styleguide@18.5.10-alpha.5547cd.0<br/>@codecademy/webpack-config@5.2.1-alpha.5547cd.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5fbba16eec1893a28a2da9d7--gamut-preview.netlify.app>https://5fbba16eec1893a28a2da9d7--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/5fbba16eec1893a28a2da9d7>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-icons@3.11.1-alpha.27f88a.0<br/>@codecademy/gamut-labs@5.4.2-alpha.27f88a.0<br/>@codecademy/gamut-styles@7.4.5-alpha.27f88a.0<br/>@codecademy/gamut-system@0.1.4-alpha.27f88a.0<br/>@codecademy/gamut@17.3.10-alpha.27f88a.0<br/>@codecademy/markdown-overrides@0.3.13-alpha.27f88a.0<br/>@codecademy/styleguide@18.5.10-alpha.27f88a.0<br/>@codecademy/webpack-config@5.2.1-alpha.27f88a.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5fd27729881c8941dfa7f912--gamut-preview.netlify.app>https://5fd27729881c8941dfa7f912--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/5fd27729881c8941dfa7f912>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-icons@4.3.1-alpha.afe04f.0<br/>@codecademy/gamut-illustrations@0.9.2-alpha.afe04f.0<br/>@codecademy/gamut-kit@0.2.25-alpha.afe04f.0<br/>@codecademy/gamut-labs@13.0.1-alpha.afe04f.0<br/>@codecademy/gamut-styles@9.0.1-alpha.afe04f.0<br/>@codecademy/gamut-system@0.5.2-alpha.afe04f.0<br/>@codecademy/gamut-tests@2.2.4-alpha.afe04f.0<br/>@codecademy/gamut@28.0.1-alpha.afe04f.0<br/>@codecademy/markdown-overrides@0.5.11-alpha.afe04f.0<br/>@codecademy/styleguide@32.0.2-alpha.afe04f.0<br/>@codecademy/variance@0.1.1-alpha.afe04f.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://6054b576f0eb57385046a1a5--gamut-preview.netlify.app>https://6054b576f0eb57385046a1a5--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/6054b576f0eb57385046a1a5>Deploy Logs</a></p>

**Review by jakemhiller — APPROVED** [review]

**Pull request merged**